ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -40,10 +40,7 @@ cluster.name: {{ elasticsearch_cluster_name }}
 # from configuring them manually. You can tie this node to a specific name:
 #
 # node.name: "Franz Kafka"
-{% if elasticsearch_node_name %}
-{#- todo: this only makes any sense for single machine setup -#}
-node.name: "{{ elasticsearch_node_name }}"
-{% endif %}
+node.name: "elasticsearch_node_{{ host_name }}"
```

Comment:
I think this shouldn't have a prefix, so you can just specify the whole thing as an inventory variable `es_node_name` (on prod it's `hqes0` and `hqes1`)